pkg/config/templates/wicked-setup-bridge.sh

```diff
@@ -7,11 +7,14 @@ case $ACTION in
 	pre-up)
 		# enable vlan-aware
 		ip link set $INTERFACE type bridge vlan_filtering 1
-		;;
-
-	post-up)
+		# reset to default vlan state
 		# accept all vlan, PVID=1 by default
 		bridge vlan add vid 2-4094 dev $INTERFACE self
```

Comment:
A general question: If we're going to drop the VLAN sub-interface for the node's management interface, can we get rid of this all-range-vlan-addition command?

```
bridge vlan add vid 2-4094 dev $INTERFACE self
```

Reply:
Thanks for providing the context. Now I know it is for the declarative outcome.

I was thinking if we could change this line into the exact opposite:

```
bridge vlan del vid 2-4094 dev $INTERFACE self
```

Will that achieve the same goal (resetting VLANs for the `mgmt-br` interface) and eliminate those unnecessary VIDs at the same time?

Reply:
Yup. It means the same thing. Actually enable vlan_filtering can allow all VLAN packet to pass through.